docs/src/components/CodeBlock.vue

```diff
@@ -1,5 +1,8 @@
 <template>
   <div class="code-block" :class="rootClass">
+    <Icon v-if="inline" name="chevron-right" rootClass="code-block__prompt" iconClass="icon_size_sm foreground-lighter" />
+    <pre tabindex="0" v-if="inline" class="pre"><code><slot /></code></pre>
+    <pre tabindex="0" v-else class="pre"><slot /></pre>
```

Comment:
Moved the copy button to appear after the code block and also made the code block focusable.